django/http/request.py

```diff
@@ -230,6 +288,17 @@ def body(self):
         if not hasattr(self, '_body'):
             if self._read_started:
                 raise RawPostDataException("You cannot access body after reading from request's data stream")
+
```

Comment:
@edevil You're correct, by my reading.
Both #2403 and #3020 protected `.body` against exceeding `DATA_UPLOAD_MAX_MEMORY_SIZE` (and a separate check against multipart parsing that accesses the stream instead). Unclear to me why the change in tack here?